functions.sh

```diff
@@ -206,6 +206,18 @@ build_env() {
             sed -i -e "s/{PG_USER}/${PG_USER}/" settings/prod.py
         popd
 
+        pushd "${TARGET}"/orion
+            cp settings/prod.py.template settings/prod.py
```

Comment:
@karpa4o4 (cc @mdanil007 )
Can you please have a look at this https://pypi.org/project/python-dotenv/ ?
I wonder if we can read read variables from .env file and get rid of settings/prod.py.template in orion?
What do you think?